layers/+completion/helm/README.org

```diff
@@ -171,15 +173,29 @@ Initiate the transient state with ~M-SPC~ or ~s-M-SPC~ while in a =Helm= buffer.
 | ~T~                  | mark all candidates                                  |
 | ~v~                  | execute persistent action                            |
 
+** Files
+In the =helm-files= buffer:
+
+| Key binding  | Description                            |
+|--------------+----------------------------------------|
+| ~S-<return>~ | open the selected file in other window |
+
+** Buffers
+In the =helm-buffers= buffer:
+
+| Key binding  | Description                              |
+|--------------+------------------------------------------|
+| ~S-<return>~ | open the selected buffer in other window |
+
 ** Bookmarks
 In the =helm-bookmarks= buffer:
 
-| Key binding | Description                                  |
-|-------------+----------------------------------------------|
-| ~C-d~       | delete the selected bookmark                 |
-| ~C-e~       | edit the selected bookmark                   |
-| ~C-f~       | toggle filename location                     |
-| ~C-o~       | open the selected bookmark in another window |
+| Key binding  | Description                                |
+|--------------+--------------------------------------------|
+| ~C-d~        | delete the selected bookmark               |
+| ~C-e~        | edit the selected bookmark                 |
+| ~C-f~        | toggle filename location                   |
```

Comment:
t seems more intuitive if this is the only toggle we need

inspired by https://github.com/emacs-evil/evil-collection/blob/496550d8dd53a057cfb39d5f0c2b9d628e5c3a72/modes/bookmark/evil-collection-bookmark.el#L56